src/System Application/App/Extension Management/src/ExtensionSettings.Page.al

```diff
@@ -117,6 +117,26 @@ page 2511 "Extension Settings"
                     ToolTip = 'Specifies the commit ID of the source code for the current version of the project.';
                 }
             }
+
+            group("Build app metadata")
+            {
+                Caption = 'Build app metadata';
+                Visible = IsBuildInformationAvailable;
+
+                field(BuildBy; BuildBy)
+                {
+                    Caption = 'Build by';
+                    Editable = false;
+                    ToolTip = 'Specifies the name of the system that assisted in the compilation of this extension.';
+                }
+                field(BuildUrl; BuildUrl)
+                {
+                    Caption = 'Build URL';
+                    Editable = false;
+                    ToolTip = 'Specifies the URL of where the package can be found.';
```

Comment:
```suggestion
                    ToolTip = 'Specifies the URL where the package can be found.';
```

Reply:
Also updated descriptions to match app.json ones.